src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -350,7 +362,30 @@ private function autocomplete(OutputInterface $output, Question $question, $inpu
         // Reset stty so it behaves normally again
         shell_exec(sprintf('stty %s', $sttyMode));
 
-        return $ret;
+        return $fullChoice;
+    }
+
+    /**
+     * Determine the most recent value the user entered.
+     *
+     * @param $entered
+     *
+     * @return string
+     */
+    private function mostRecentlyEnteredValue($entered)
+    {
+        $tempEntered = $entered;
+
+        if (false !== strpos($entered, ',')) {
+            $choices = explode(',', $entered);
+            $lastChoice = trim($choices[\count($choices) - 1]);
+
+            if (\strlen($lastChoice) > 0) {
+                $tempEntered = $lastChoice;
+            }
+        }
```

Comment:
Can you inline this logic in `autocomplete()` with a doc comment?

Reply:
Thanks @chalasr!
Could you please explain what you mean? I moved that logic into its own function because it's used twice in `autocomplete()`

Reply:
Fair enough :)